x-pack/platform/test/cases_api_integration/common/users.ts

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
```

Comment:
copied to simplify imports. we will removed original files in the follow-up with more tests relocated